apps/scroll-cd/src/app/listen-outside-zone.util.ts

```diff
@@ -0,0 +1,16 @@
+import { getZoneUnPatchedApi } from '@rx-angular/cdk/internals/core';
+
+export function listenOutsideZone(
+  target: HTMLElement | Window | Document,
+  type: string,
+  listener: EventListener
+): () => void {
+  getZoneUnPatchedApi(target, 'addEventListener').call(target, type, listener);
+  return () => {
+    getZoneUnPatchedApi(target, 'removeEventListener').call(
+      target,
+      type,
+      listener
+    );
+  };
+}
```

Comment:
question: why do you need to call getZoneUnPatchedApi twice ?

Reply:
Thanks for accepting the solution :). `removeEventListener` is also patched by zone. So if you want to go all-in zone-less it's better to unpatch removal also.

Native 

![remove-patched](https://user-images.githubusercontent.com/15088626/209958708-df168669-7fc4-415b-8835-6078f3e2d878.png)

Unpatched

![remove-unpatched](https://user-images.githubusercontent.com/15088626/209958748-b9141804-0bd8-4533-85a2-bb0317a5e795.png)

Reply:
I'm so tired, I didn't see that this was two different listeners 😅